framework/atoms/messages/_messages.scss

```diff
@@ -53,97 +48,73 @@ $messages-text-size: $font-size-little;
     font-weight: bold;
   }
 
-  ul,
   p {
     max-width: 95%;
     margin-bottom: 0;
     font-size: $messages-text-size;
   }
 
-  li {
-    list-style-type: none;
-  }
-
   > :last-child {
     padding-bottom: 0;
   }
-}
 
-.messages__list {
-  max-width: auto;
-  margin-bottom: 0;
-  padding-left: 0;
-  font-family: $font-family-sans-serif;
+  .close {
+    display: none;
+  }
 }
 
-.messages__list-item {
-  margin-top: 0.5em;
+// Bootstrap overrides.
+.alert {
+  margin: 1.667em 0;
+  border-radius: 0;
+
+  .close {
```

Comment:
Can't notice that this is too generic and will need to have better naming later.